scripts/tiqit/database.py

```diff
@@ -32,7 +32,7 @@
 
 import tiqit
 import sqlite3 as sqlite, os.path
-import cPickle as pickle
+import pickle as pickle
```

Comment:
```suggestion
import pickle
```

Reply:
Worth checking if this has happened in any other files as I may not have spotted it.